test/e2e/smoketest.go

```diff
@@ -138,7 +138,7 @@ func executeSmokeTest(apiTracesEndpoint, collectorEndpoint string, hasInsecureEn
 		bodyString := string(bodyBytes)
 
 		// The first requests to newly created ES might fail
-		if !strings.Contains(bodyString, "errors\":null") && !strings.Contains(bodyString, "all shards failed") {
+		if !strings.Contains(bodyString, "errors\":null") {
```

Comment:
We should not tolerate returning `all shards failed` from the query service. This was a bug in ES operator which has been fixed.